python/cuspatial/pyproject.toml

```diff
@@ -36,6 +36,7 @@ dependencies = [
     "libcuspatial==25.4.*,>=0.0.0a0",
     "numpy>=1.23,<3.0a0",
     "rmm==25.4.*,>=0.0.0a0",
+    "shapely<2.1.0",
```

Comment:
I think the `conda` recipe should pick up a `run_constrained` entry with this same pin, to achieve similar behavior.

Otherwise, users installing the `cuspatial` conda package will get the newer versions of `shapely` and face the issues this PR aims to fix.

You're not seeing that show up in CI because of the `shapely` pin being added in `dependencies.yaml`.

Reply:
Good catch, added a `run_constrained` along with a link to #1561